pyproject.toml

```diff
@@ -29,7 +29,7 @@ classifiers = [
 ]
 
 [project.optional-dependencies]
-dev = ["flake8", "ruff == 0.4.1", "pytest"]
+dev = ["flake8", "ruff == 0.4.2", "pytest"]
```

Comment:
Given that `ruff` is a superset can we lose `flake8`?
Should we just unpin `ruff`?